requirements.txt

```diff
@@ -1,4 +1,4 @@
-Babel==0.9.6
+Babel==2.5.3
```

Comment:
Note: *Blocker* This installation / update will have to happen on production *prior* to merge! Our `olsystem` provisioning doesn't re-install / upgrade pip dependencies (I don't believe). It *could* (we have a method in our fabfile) but we're not doing this yet in our `deploy`

Reply:
Right, I thought there might be issues with this. I don't think that Babel 2.5.3 is strictly _necessary_ for getting messages.pot updated, although I did update early because I thought it would improve my chances for getting things running smoothly.

It might be nice to do the update anyway, since there have been so many changes since the old version.  `scripts/i18n-messages` does not really produce useful output without these changes anyway, so I belive production Babel could be updated first without breaking anything that isn't already broken!

In dev I changed the line to `Babel>=0.9.6` and installed the latest manually while I was experimenting. Note there is also `test-requirements.txt` which has it too.

Reply:
`Babel==2.5.3` has been venv installed on `ol-www1`, `ol-web3`, `ol-web4`, and `ol-home`.